docs/AboutUs.adoc

```diff
@@ -19,8 +19,8 @@ Responsibilites: Logic, Model
 '''
 
 === Ang Yee Chin
-image::ANG-YC.png[width="150", align="left"]
-{empty}[http://github.com/ANG-YC[github]] [<<#, portfolio>>]
+image::Ang-YC.png[width="150", align="left"]
+{empty}[http://github.com/Ang-YC[github]] [<<#angyeechin, portfolio>>]
```

Comment:
It should be `[<<angyeechin#, portfolio>>]` instead!

Reply:
Thanks!